ajax/central.php

```diff
@@ -69,14 +69,16 @@
 switch ($_REQUEST['widget']) {
     case 'central_count':
         if (method_exists($itemtype, 'showCentralCount')) {
-            $itemtype::showCentralCount($params['foruser'] ?? false);
+            $static = new $itemtype();
```

Comment:
I'm not sure I understand why we have to instantiate a class to call one of its static method

Reply:
(previous remark is the same for all changes of that type.)

Also, I'm not fan of the name of that variable, something like `$instance` or `$item` would make more sense IMHO)

Reply:
This is because `$itemtype` is a string.
I wasn't inspired by the name, so I'll take the original name and add `_class`. I've seen a case elsewhere where it was done like that.

Reply:
I still do not see why an instanciation is required; it seems to much to fix a phpstan issue

Reply:
What should I do in this case?

Reply:
What was the initial issue reported by PHPStan ?

Reply:
> What was the initial issue reported by PHPStan ?

```
Static call to instance method stdClass::showCentralCount().
```

Reply:
This seems to do the trick:
```
/** @var class-string $itemtype */
$itemtype = $_REQUEST['itemtype'];
```

Reply:
Should probably do a is_string check to validate the input.